tests/Fixer/Strict/DeclareStrictTypesFixerTest.php

```diff
@@ -141,11 +155,11 @@ public function provideMessyWhitespacesCases()
     {
         return [
             [
-                "<?php declare(strict_types=1);\r\nphpinfo();",
+                "<?php\r\ndeclare(strict_types=1);\r\n\tphpinfo();",
```

Comment:
> Oh, I didn't realise that. Should I bug this then?

no, i have no idea what is `dev-feature/braces/position_after_return_type_hint` version you had installed.
other thing - assume it was regular 2.x release. If we were suppose to fix sth, but for some reason we didn't fix it, and with newer release we catch it and finally fix it - it's expected bugfix. if we claim to fix in one way (eg "same line"), and then out of the blue redefine the rule to fix code another way (eg "new line"), then this is not bugfix but different rule under same name. this is not ok unless configured.

> I'll change the default on the position fixer to same in that case.

Position fixer is new, so it could have any default, as long as it's not conflicting with another rule.

----------------------

thing here. existing fixer, with default config, shall work same way as on master. that's it.